packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-message-list-fetch.service.ts

```diff
@@ -74,9 +77,30 @@ export class MessagingMessageListFetchService {
         },
       };
 
+      const datasource = await this.twentyORMManager.getDatasource();
+
+      await this.syncMessageFoldersService.syncMessageFolders({
+        workspaceId,
+        messageChannelId: messageChannel.id,
```

Comment:
we actually already have the messageChannel with connected account, let's pass it here :)